.azure-pipelines.yml

```diff
@@ -47,10 +47,10 @@ steps:
         -DopenPMD_USE_PYTHON=OFF -DBUILD_TESTING=OFF -DBUILD_EXAMPLES=OFF
     fi
     if [ "${WARPX_CI_RZ_OR_NOMPI:-FALSE}" == "TRUE" ]; then
-      cmake-easyinstall --prefix=/usr/local git+https://bitbucket.org/icl/blaspp.git \
-        -Duse_openmp=OFF -Dbuild_tests=OFF
+      cmake-easyinstall --prefix=/usr/local git+https://bitbucket.org/axelhuebl/blaspp.git \
```

Comment:
Hopefully we can continue with the upstream repo once this PR is merged: https://bitbucket.org/icl/blaspp/pull-requests/19